src/routing/pages/Home.tsx

```diff
@@ -20,6 +20,17 @@ export const Home = () => {
           </CardAction>
         </CardContent>
       </Card>
+      <Card>
+        <CardHeader>"Compound components" nativos - Gonza</CardHeader>
```

Comment:
NIT: En vez de usar comillas podria ponerse en negrita

```suggestion
        <CardHeader><b>Compound components</b> nativos - Gonza</CardHeader>
```